src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
+const Home = props => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="d-flex flex-column justify-content-center ">
+      <section className="align-self-center text-center">
+        <h1> {"{...}"} </h1>
+        <h2> Hello Coder ! </h2>
+        <h3> Are you ready for the next challenge ?
+        <p> Take a Quiz </p>
```

Comment:
For lines 11-13, it feels more like a single paragraph — let's change it to be a single paragraph

Also, headings should not have `p`s inside them since headings already display sentences

Reply:
I added that to get a break line, Thanks for correction.